01-introduction-geospatial-data.ipynb

```diff
@@ -36,7 +36,7 @@
     "\n",
     "We can use the GeoPandas library to read many of those GIS file formats (relying on the `fiona` library under the hood, which is an interface to GDAL/OGR), using the `geopandas.read_file` function.\n",
     "\n",
-    "For example, let's start by reading a shapefile with all the countries of the world (from http://www.naturalearthdata.com/downloads/110m-cultural-vectors/110m-admin-0-countries/), and inspect the data:"
+    "For example, let's start by reading a shapefile with all the countries of the world (from https://www.naturalearthdata.com/http//www.naturalearthdata.com/download/110m/cultural/ne_110m_admin_0_countries.zip), and inspect the data:"
```

Comment:
I changed the wording here a little bit, as the participants are not expected to download it themselves, the data will be provided in the `/data` directory (I added them earlier today)